### DISCUSSION TITLE
提问关于docker下配置texlive环境

### DISCUSSION THREAD

**Discussion by Excespo** [sjtug/SJTUThesis#815] [提问] [answered]
您好，最近做作业想用docker来配置环境，拉取的是ubuntu18.04的镜像，如果直接`apt-get install texlive`安装的是texlive2019，然后会出现[这个错误](https://github.com/sjtug/SJTUThesis/discussions/693)，无法编译模板。我想知道有办法安装指定版本的texlive嘛？
实际上我在[sjtu镜像站ctan](https://mirrors.sjtug.sjtu.edu.cn/ctan/systems/texlive/Images/)里面找到了texlive2022，然后`wget`到本地了，但是不知道怎么只在docker内安装。
谢谢！

**Comment by Excespo** [comment]
勘误，拉取的是ubuntu20.04而不是18.04，抱歉

**Comment by AlexaraWu** [comment, marked as answer]
挂载 ISO 文件，使用 `install-tl` 安装。

话说直接拉现成的镜像不是更方便？

FYI: https://hub.docker.com/r/texlive/texlive/

**Reply by Excespo** [reply]
谢谢回复！确实拉取现成的比较方便，已经能编译通了，谢谢！
